recipe/meta.yaml

```diff
@@ -10,10 +10,8 @@ source:
   sha256: b47e54282ef13cddffdb00aea9183a87175a2372c91a915259086a3f444c42f4
 
 build:
-  number: 1001
-  script: python -m pip install --no-deps --ignore-installed .
-  # Cython issues
-  skip: True  # [py>=37]
+  number: 1002
+  script: USE_CYTHON=1 {{ PYTHON }} -m pip install --no-deps --ignore-installed .
```

Comment:
Appveyor builds fail with `'USE_CYTHON' is not recognized as an internal or external command,
operable program or batch file.`, see https://ci.appveyor.com/project/conda-forge/biom-format-feedstock/builds/21535902/job/vauvfd0x6u3xjhnq#L539